pkg/pillar/containerd/oci.go

```diff
@@ -315,20 +315,24 @@ func (s *ociSpec) UpdateFromDomain(dom *types.DomainConfig) {
 // OCI runtime spec takes precedence). In addition to that each volume is
 // expected to have rootfs subfolder with a full rootfs filesystem
 func (s *ociSpec) UpdateFromVolume(volume string) error {
+	//if not based on an OCI image, nothing to update here
+	if volume == "" {
+		return nil
+	}
```

Comment:
Doesn't the caller already check this??

Reply:
No, and I hit that problem during my experiments with VMs